src/components/App/App.js

```diff
@@ -2,34 +2,52 @@ import React, { Component } from 'react';
 import Header from '../Header/Header';
 import Movies from '../Movies/Movies';
 import Film from '../Film/Film';
-import movieData from './movie-data';
 import './App.css';
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
-      movies: movieData.movies,
-      currentMovie: {},
-      isHome: true
+      movies: [],
+      currentMovie: '',
+      isHome: true,
+      isLoading: true,
+      error: false
+    }
+  }
+
+  componentDidMount() {
+    fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies/')
```

Comment:
You'll probably want to pull this (and the other network request functions) out into its own apiCalls.js file